### PR TITLE
Add level manager

### PR DESCRIPTION
**Pull request by mmcleod89**
Level management script added. 

Introduces a clock which counts down as long as the bool `gameActive` is true (which can be used to stop the timer at the end of the game and also pause the timer during the help) 

Molecules & relative frequency lists added, and methods to generate random molecules. 

Dummy molecule prefabs used to check that molecules appear but these can be removed from the project when real molecules are available. 

Also adjusted the grid to the correct ratio - we were double correcting now because the defaults in this version of unity are different from the ones in the version I created the test in, and so the cell width was already 0.86!

### CODE REVIEW

File: Unity_Project_Files/Aligne-d/Assets/Scripts/Level_Management.cs [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+
+    private void GeneratePreviews()
+    {
+        Debug.Log("GenPreviews");
+        foreach (var marker in previewMarkers)
+        {
+            Vector3 position = marker.transform.position;
+            Debug.Log(position);
+            int index = Random.Range(0, numMoleculesAvailable);
+            GameObject newMolecule = Instantiate<GameObject>(moleculesAvailable[index]);
```
Comment (tcouch) [inline Unity_Project_Files/Aligne-d/Assets/Scripts/Level_Management.cs:62]: This looks like we're not accounting for molFrequencies yet. Is that right?

**Review by pineapple-cat — APPROVED** [review]
Gave it a test run and the clock seems to be working well, though the molecule previews were invisible on the board for me. Could be an issue on my end though since manually giving them a sprite while the game was running makes it evident that they are showing up in the expected places.

**Comment by mmcleod89** [comment]
Thanks both; have updated PR to utilise molecule frequencies. 
@pineapple-cat Can you check if this updated version still gives you weird stuff? It might be that I didn't save my scene properly so a bunch of serialized fields weren't hooked up.

**Comment by pineapple-cat** [comment]
Yes everything works now; the serialized fields were empty before and I didn't find the correct available molecule prefabs.

**Pull request merged**